js/src/initializers/admin.js

```diff
@@ -121,7 +121,7 @@ export default function initAdmin( jQuery ) {
 	 */
 	function wpseoCopyHomeMeta() {
 		jQuery( "#copy-home-meta-description" ).on( "click", function() {
-			jQuery( "#open_graph_frontpage_desc" ).val( jQuery( "#meta_description" ).val() );
+			jQuery( "#og_frontpage_desc" ).val( jQuery( "#meta_description" ).val() );
```

Comment:
This is the only change that may need testing, but needs the changes in P3-594 for that. The PR for P3-594 will include the change and test instructions.